UberBarber/UberBarber/View/UserRegistration.xaml

```diff
@@ -219,8 +221,8 @@
                                         AlignmentX="Left"/>
                         </TextBox.Background>
                     </TextBox>
-                    <!-- Label with error info-->
-                    <Label x:Name="LabelInfoUserReg" FontFamily="Montserrat" Foreground="Red" FontSize="12" HorizontalAlignment="Center"/>
+                    <!-- TextBlock with error info-->
+                    <TextBlock x:Name="TextBlockInfoUserReg" FontFamily="Montserrat" Margin="0,5,0,-5" Foreground="Red" FontSize="12" HorizontalAlignment="Center" TextWrapping="Wrap"/>
```

Comment:
Keep the code "clean". X:name, FontFamily, ForeGround... write one after the other

Reply:
Corrected